guides/common/modules/proc_integrating-infoblox-dhcp.adoc

```diff
@@ -1,7 +1,10 @@
 [id="integrating-infoblox-dhcp"]
 = Integrating Infoblox DHCP
 
-Install the DHCP Infoblox provider on {ProductName}.
+If you have an Infoblox DHCP service in your network, you can integrate this service into {ProjectServer}.
+The integration enables you to continue using your existing DHCP server, and {Project} manages IP leases and boot configurations on the DHCP server during the provisioning of hosts.
+
+With this type of integration, {Project} uses the Infoblox API to authenticate and manage DHCP leases in local or remote Infoblox DHCP instances.
```

Comment:
Infoblox is never local because it's always on a different machine (since it's an appliance). Note it can be machines as well since it can run HA.

We should mention we use the Infoblox Web API (WAPI). As far as I can see this is their name for their REST API (as opposed to their deprecated Perl API or PAPI).